boot/boot.js

```diff
@@ -1444,6 +1444,7 @@ $tw.loadTiddlersBrowser = function() {
 	for(var t=0; t<containerIds.length; t++) {
 		$tw.wiki.addTiddlers($tw.wiki.deserializeTiddlers("(DOM)",document.getElementById(containerIds[t])));
 	}
+	callback();
```

Comment:
> The main question is, why are you providing an asynchronous interface for a synchronous flow?

If this is the main question, I better answer it 😄 Because I want to be able to make it asynchronous if I want to without directly editing boot.js. You can see what I mean if you look at the end of the file in the link below.

https://github.com/Arlen22/TiddlyWiki5/blob/patch-7/boot/boot-node.js

This code can either be called in boot.js, or in the file that `require`d boot.js, so the real point of this pull request is that it is from the perspective of `$tw.boot.startup` wanting to not care whether the `loadTiddlersNode` and `loadTiddlersBrowser` are async or not.